### PR TITLE
update evm info to mainnet info

### PR DESCRIPTION
**Pull request by bthaile**
change testnet evm information to mainnet information.

### CODE REVIEW

File: docs/evm/using.mdx [hunk 1]
```diff
@@ -34,10 +34,10 @@ Manual method: Add EVM on Flow as a custom network to MetaMask:
 | --------------- | ------------------------------------ |
 | Network Name    | Flow Testnet                         |
 | Description     | The public RPC url for Flow Testnet  |
-| RPC Endpoint    | https://testnet.evm.nodes.onflow.org |
-| Chain ID        | 545                                  |
+| RPC Endpoint    | https://mainnet.evm.nodes.onflow.org |
```
Comment (gregsantos) [inline docs/evm/using.mdx:37]: this should be testnet url? Network Name is "Flow Testnet"

**Review by sisyphusSmiling — APPROVED** [review]

**Review by gregsantos — CHANGES_REQUESTED** [review]

File: docs/evm/using.mdx (hunk 1 above)
Comment (gregsantos) [reply docs/evm/using.mdx:37]: oh I see, I guess you just need to change Network Name to Mainnet

Comment (gregsantos) [reply docs/evm/using.mdx:37]: I think it's just supposed to be "Flow" Can we confirm with Jerome

Comment (bthaile) [reply docs/evm/using.mdx:37]: I missed updating the name, I thought we would want to point ppl to mainnet, now that its live

File: src/components/addNetworkButton.tsx [hunk 2]
```diff
@@ -36,9 +36,9 @@ export const AddNetworkButton = () => {
         method: 'wallet_addEthereumChain',
         params: [
           {
-            chainId: '0x221',
-            chainName: 'Flow Testnet',
-            rpcUrls: ['https://testnet.evm.nodes.onflow.org'],
+            chainId: '0x2eb', // 747 in hexadecimal
+            chainName: 'Flow Mainnet',
```
Comment (gregsantos) [inline src/components/addNetworkButton.tsx:40]:
```suggestion
            chainName: 'EVM on Flow',
```

**Review by gregsantos — CHANGES_REQUESTED** [review]

**Review by gregsantos — APPROVED** [review]

**Pull request merged**